### PR TITLE
Fixed twitter-x icon

### LINKED ISSUE #1507: [Suggestion] : Change 'Twitter' icon on home page to 'X'
The Twitter icon on the home page is still the old bird icon, even though Twitter has updated their branding to use the X icon. Please update the home page to use the new X icon.

image

Can you assign this issue to me?

### PR DESCRIPTION
**Pull request by khanjanSanghavi**
### Fixes the old twitter logo to new one

> Fixes #1507 

### Select what type of change this PR introduces:

1. -[x] **Just code/docs improvement** (no functional change). 
2. -[x] **Bug fix** (non-breaking change which fixes an issue).
3. [ ] **New feature** (non-breaking change which adds functionality).
4. [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

Screenshot (84)

### CODE REVIEW

File: web/src/components/Nav/SocialIcons.js [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
 )
 
 export const TwitterIcon = () => (
-  <svg width='24' height='24' fill='currentColor' viewBox='0 0 335 276'>
-    <title>Twitter</title>
-    <path d='m302 70a195 195 0 0 1 -299 175 142 142 0 0 0 97 -30 70 70 0 0 1 -58 -47 70 70 0 0 0 31 -2 70 70 0 0 1 -57 -66 70 70 0 0 0 28 5 70 70 0 0 1 -18 -90 195 195 0 0 0 141 72 67 67 0 0 1 116 -62 117 117 0 0 0 43 -17 65 65 0 0 1 -31 38 117 117 0 0 0 39 -11 65 65 0 0 1 -32 35'></path>
+  // <svg width='24' height='24' fill='currentColor' viewBox='0 0 335 276'>
+  //   <title>Twitter</title>
+  //   <path d='m302 70a195 195 0 0 1 -299 175 142 142 0 0 0 97 -30 70 70 0 0 1 -58 -47 70 70 0 0 0 31 -2 70 70 0 0 1 -57 -66 70 70 0 0 0 28 5 70 70 0 0 1 -18 -90 195 195 0 0 0 141 72 67 67 0 0 1 116 -62 117 117 0 0 0 43 -17 65 65 0 0 1 -31 38 117 117 0 0 0 39 -11 65 65 0 0 1 -32 35'></path>
+  // </svg>
```
Comment (Martinsos) [inline web/src/components/Nav/SocialIcons.js:14]: Let's remove this commented out part of the code!

File: web/src/components/Nav/SocialIcons.js [hunk 2]
```diff
@@ -1,27 +1,37 @@
...
+  <svg
+    width="24"
+    height="24"
+    fill="currentColor"
+    class="bi bi-twitter-x"
+    viewBox="0 0 16 16"
+  >
```
Comment (Martinsos) [inline web/src/components/Nav/SocialIcons.js:21]: All the other icons have `<title>` tag, let's also add one here.

**Review by Martinsos — COMMENTED** [review]
Looking good, but please check out the two comments I left, once those are good we can merge!

**Comment by khanjanSanghavi** [comment]
@Martinsos  Done, Please have a look

**Review by Martinsos — APPROVED** [review]

**Comment by Martinsos** [comment]
@khanjanSanghavi looking good, merging, thanks!

**Pull request merged**